mindsdb/api/http/namespaces/config.py

```diff
@@ -214,14 +218,27 @@ def get(self, dependency):
         else:
             return f'Unkown dependency: {dependency}', 400
 
+        outs = b''
+        errs = b''
         try:
-            sp = subprocess.Popen(['pip3', 'install', *dependency])
-            sp.wait()
-        except Exception:
-            try:
-                sp = subprocess.Popen(['pip', 'install', *dependency])
-                sp.wait()
-            except Exception:
-                return 'Failed to install', 400
+            sp = subprocess.Popen(
+                [sys.executable, '-m', 'pip', 'install', *dependency],
```

Comment:
Should we add pip3 as we did previously or it will be assumed by sys.executable if it is e.g python3?

Reply:
@ZoranPandovski no, here 'pip' executed as module for particular python. In that case - the same python as app python.